CMakeLists.txt

```diff
@@ -270,15 +263,17 @@ endif(TT_METAL_BUILD_TESTS OR TTNN_BUILD_TESTS)
 # For top level install: cmake --build build --target install  or  make/ninja install -C build
 ############################################################################################################################
 # Install for build artifacts that will upload build/lib
+include(GNUInstallDirs)
+
 install(TARGETS tt_metal
     ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
     LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
-    COMPONENT tt_build_artifacts
+    COMPONENT dev
```

Comment:
Do we use the `dev` component anywhere? What is it for?

Reply:
The convention is to use dev for a development package in ubuntu.

In theory we could one day have a runtime package that only supplies libraries for runtime, and a dev package which includes header files and related - for development.

However, as a first cut, if we can just get a functional development package working for developers it would be a win.